src/groups/bmq/bmqa/bmqa_closequeuestatus.h

```diff
@@ -62,28 +62,29 @@ namespace bmqa {
 class CloseQueueStatus {
   private:
     // DATA
+
+    /// queueId associated with the open
+    /// queue operation
     QueueId d_queueId;
-    // queueId associated with the open
-    // queue operation
 
+    /// Result code of the operation
+    /// (success, failure)
     bmqt::CloseQueueResult::Enum d_result;
-    // Result code of the operation
-    // (success, failure)
 
+    /// Optional string with a human
+    /// readable description of the error,
+    /// if any
     bsl::string d_errorDescription;
-    // Optional string with a human
-    // readable description of the error,
-    // if any
 
   public:
     // TRAITS
     BSLMF_NESTED_TRAIT_DECLARATION(CloseQueueStatus, bslma::UsesBslmaAllocator)
 
     // TYPES
 
-    // Use of an `UnspecifiedBool` to prevent implicit conversions to
-    // integral values, and comparisons between different classes which
-    // have boolean operators.
+    /// Use of an `UnspecifiedBool` to prevent implicit conversions to
```

Comment:
I'm not sure this was meant as public documentation, but I'm okay with it being public documentation.

Reply:
This one is totally fine to keep internal, seems harmless to do so.